torchtitan/experiments/flux/train.py

```diff
@@ -112,32 +112,33 @@ def train_step(self, input_dict: dict[str, torch.Tensor], labels: torch.Tensor):
 
         bsz = labels.shape[0]
```

Comment:
Why is `labels` on CPU? Shouldn't it be on GPU? Then you won't need `with torch.device(self.device)` and the extra indention.